### PR TITLE
Add Koios Python builder tool

### PR DESCRIPTION
**Pull request by QuixoteSystems**
---

## Showcase addition

* Name: *Koios Python*
* Description: *Koios Python wrapper which allow interacting with all information and parameters stored on the Cardano blockchain.*
* Link:   [Koios Python](https://github.com/cardano-community/koios-python)
* Source: [Koios Python](https://github.com/cardano-community/koios-python)
* Tags:
  * `analytics`
  * `explorer`
  * `opensource`

### CODE REVIEW

**Review by rphair — COMMENTED** [review]
thanks... as per #807, can you:

- fix the merge conflict by merging your branch with current `staging` and/or
    - removing the `api` tag which is obsoleted in the current `staging` build?
    - adding a `,` to the end of the `Showcases` array (i.e. at the end of your own added code)?
- add a featured image as you did for your other project?  Maybe something not generated from fixed width text that clips at the image border?

**Comment by QuixoteSystems** [comment]
I just add a fancy logo for the project. 😆

**Comment by rphair** [comment]
Last commit was to fix an improperly resolved merge conflict that left one or two elements in `Showcases` without the delimiting comma.

**Comment by rphair** [comment]
GitHub ate my top level review comment (I swear) so here it is again: after fixing some hairy merge conflicts this builds OK with current `staging` ✔️

**Review by fill-the-fill — APPROVED** [review]

File: src/data/builder-tools.js [hunk 1]
```diff
@@ -769,6 +769,14 @@ export const Showcases = [
...
     getstarted: null,
     tags: ["operatortool", "api"],
+  },
+  {
+    title: "Koios Python",
+    description: "Koios Python wrapper which allow interacting with all information and parameters stored on the Cardano blockchain.",
+    preview: require("./builder-tools/koios-python.png"),
+    website: "https://github.com/cardano-community/koios-python",
+    getstarted: null,
+    tags: ["python", "api", "sdk"],
```
[the remark this answers is not in the document]
Comment (fill-the-fill) [reply src/data/builder-tools.js]: Not sure why Github mentions the resolved issue

**Comment by QuixoteSystems** [comment]
I resolved the issue, I don't know why I had to resolve it.

**Comment by rphair** [comment]
thanks @fill-the-fill - I would like to know how you did that merge into the submitter's branch ☝️ (I've been looking for a way of doing that; rather than resolving the merge conflicts by hand).

Also @fill-the-fill since you approved this with the `operatortool` tag, if you let me know how or why this could be considered an Operator Tool then please let me know & I'll approve it as well 🤓

**Comment by fill-the-fill** [comment]
> thanks @fill-the-fill - I would like to know how you did that merge into the submitter's branch ☝️ (I've been looking for a way of doing that; rather than resolving the merge conflicts by hand).
> 
> Also @fill-the-fill since you approved this with the `operatortool` tag, if you let me know how or why this could be considered an Operator Tool then please let me know & I'll approve it as well 🤓

1. At the bottom in reviewing I got a message similar to this one that it's out of date, so simple `update branch` helps. But this button doesn't always display. 
2. 
Screenshot 2022-10-21 at 16 17 39

3. I approved it having `python`, `sdk` only, `operatortool` wasn't part of it. Very confusing, a bit of a buggy PR.

**Review by rphair — APPROVED** [review]
@QuixoteSystems just so we can move forward with this I've removed the `operatortool` tag... if you want to dispute this, please give your reasons.  Also this "resolves" the last unresolved conversation which you may have been referring to in your own last comment. 😎

**Comment by rphair** [comment]
p.s. my own first review comments are now duplicated in my last review, for some reasons I don't understand.  @fill-the-fill you're right, this is a haunted PR 🤣

**Review by os11k — APPROVED** [review]

**Comment by QuixoteSystems** [comment]
@rphair is fine! Maybe you are right and is just a tool not a specific Pool Tool. No worries at all 😉

**Pull request merged**